### PR TITLE
status: 2023q2: BATMAN support for the FreeBSD kernel

### PR DESCRIPTION
**Pull request by obiwac**
2023Q2 status report for the BATMAN support on FreeBSD GSoC project.

I totally missed the deadline for submitting status reports, my apologies!

### CODE REVIEW

File: website/content/en/status/report-2023-04-2023-06/batman.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+BATMAN (Better Approach to Mobile Ad-hoc Networking), as developed and used by the Freifunk project, is a routing protocol for (primarily wireless) multi-hop ad-hoc networks.
+Freifunk is a German initiative to build an open Wi-Fi network at city-scale, based on the principles of net-neutrality.
+BATMAN's motive is to be a completely decentralized protocol; no one node in the network knows or has to care about the topology of the whole network.
+
+Support for this protocol is provided by the batman-adv kernel module on Linux, and this project aims to bring that to FreeBSD.
+This includes the kernel module itself, but also userland networking libraries and tools necessary to create BATMAN networks. 
+
+Currently, creating interfaces and interacting with them works (with both Linux and FreeBSD userspaces), and packet transmission (kind of) works, although it's incomplete as of yet.
+Support for batadv interfaces has been added to ifconfig(8) too.
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/batman.adoc:17]: Please use the `man` macro here.

```suggestion
Support for batadv interfaces has been added to man:ifconfig[8] too.
```

File: website/content/en/status/report-2023-04-2023-06/batman.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
+=== BATMAN support in the FreeBSD kernel
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/CallingTheBatmanFreeNetworksOnFreeBSD[Wiki page] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/CallingTheBatmanFreeNetworksOnFreeBSD[] +
+link:https://github.com/obiwac/freebsd-gsoc/pull/1[Source code (PR)] URL: https://github.com/obiwac/freebsd-gsoc/pull/1[]
```
Comment (lsalvadore) [inline website/content/en/status/report-2023-04-2023-06/batman.adoc:5]: Please avoid using the PR abbreviation: we use it in different contexts with different meanings (pull requests or problem report).
```suggestion
link:https://github.com/obiwac/freebsd-gsoc/pull/1[Source code (Pull Request)] URL: https://github.com/obiwac/freebsd-gsoc/pull/1[]
```

**Review by lsalvadore — CHANGES_REQUESTED** [review]
Just a few minor changes and it is ready for merging.

Thanks.

**Comment by obiwac** [comment]
Thanks for the review!

**Comment by lsalvadore** [comment]
Merged, thanks.

**Pull request closed without merging**